components/keyboard_shortcuts/keyboard_shortcuts_sequence/keyboard_shortcuts_sequence.tsx

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {FormatXMLElementFn, PrimitiveType} from 'intl-messageformat';
+import React, {memo} from 'react';
+import {useIntl} from 'react-intl';
+
+import {isMessageDescriptor, KeyboardShortcutsSeq} from '../keyboard_shortcuts';
+
+import {isMac} from '../../../utils/utils';
+import './keyboard_shortcuts_sequence.scss';
+import {ShortcutKetVariant, ShortcutKey} from '../../shortcut_key';
+
+type Props = {
+    shortcut: KeyboardShortcutsSeq;
+    values?: Record<string, PrimitiveType | FormatXMLElementFn<string, string>>;
+    hideDescription?: boolean;
+    hoistDescription?: boolean;
+    isInsideTooltip?: boolean;
+};
+
+function normalize(shortcut: KeyboardShortcutsSeq) {
```

Comment:
IMO a more descriptive name would be good.

something like `normalizeShortcutDescriptor`

Reply:
How about normalizeKeyboardShortcutDescriptor  or keyboardShortcutDescriptor IMO adding Keyboard makes it more clear